### DISCUSSION TITLE
password_changed is never called

### DISCUSSION THREAD

**Discussion by bdavidxyz** [janko/rodauth-rails#95] [General]
If the user is already connected, and ask for a password change, it works immediately, without any email sent. The flash message works, everything work.

However there is a function inside rodauth_mailer.rb, that is generated, named "password_changed". This function is never called, which is logic because this is not a scenario where  you expect an email to be sent. Maybe it should be removed from the generator ?

Thanks for the great work, I managed to covered all the (default) use cases through the cypress-rails gem.

**Comment by janko** [comment]
The default configuration enables the `change_password_notify` feature, which sends an email notifying the user the password changed. It calls `create_password_changed_email`, which the default configuration also overrides with a call to `RodauthMailer.password_changed`.

You might have disabled the `change_password_notify` feature and removed the corresponding configuration at some point, and forgot to remove `RodauthMailer.password_changed` method.
